File: Java/TheMovieDatabaseApp/app/src/main/java/balazs/bence/themoviedatabaseapp/MainActivity.java
package balazs.bence.themoviedatabaseapp;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Button getMovieByIDButton = (Button)findViewById(R.id.searchMovieByIdButton);
        getMovieByIDButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent searchIntent = new Intent(MainActivity.this, SearchMovieByID.class);
                startActivity(searchIntent);
            }
        });
    }

    @Override
    public void onClick(View view) {

    }
}

File: Kotlin/WeatherApp/app/src/main/java/balazs/bence/weatherapp/MainActivity.kt
package balazs.bence.weatherapp

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.EditText
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        var getForecastButton = findViewById<Button>(R.id.getForecastButton)
        getForecastButton.setOnClickListener {
            var moveIntent = Intent(this, ForecastActivity::class.java)
            val searchEditTextView = findViewById<EditText>(R.id.searchEditText)
            moveIntent.putExtra("searchTerm", searchEditText.text.toString())
            startActivity(moveIntent)
        }
    }
}
